dbt/README.md

```diff
@@ -494,63 +468,141 @@ We use the following pattern to determine where to define each column descriptio
 
 New models should generally be added with accompanying tests to ensure the
 underlying data and transformations are correct. For more information on
-testing, see [📝 How to add and run tests](#-how-to-add-and-run-tests).
+testing, see [🧪 How to add and run QC tests and reports](#-how-to-add-and-run-qc-tests-and-reports).
 
-## 📝 How to add and run tests
+## 🔨 How to rebuild models using GitHub Actions
 
-We test our data and our transformations using [dbt
-tests](https://docs.getdbt.com/docs/build/tests). We prefer adding tests
-inline in `schema.yml` config files using [generic
-tests](https://docs.getdbt.com/best-practices/writing-custom-generic-tests),
-rather than [singular
-tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests).
+GitHub Actions can be used to manually rebuild part or all of our dbt DAG.
+To use this functionality:
 
-### Data tests vs. unit tests
+- Go to the `build-and-test-dbt` [workflow page](https://github.com/ccao-data/data-architecture/actions/workflows/build_and_test_dbt.yaml)
+- Click the **Run workflow** dropdown on the right-hand side of the screen
+- Populate the input box following the instructions below
+- Click **Run workflow**, then click the created workflow run to view progress
 
-There are two types of tests that we might consider for a model:
+The workflow input box expects a space-separated list of dbt model names or selectors.
+Multiple models can be passed at the same time, as the input box values are
+passed directly to `dbt build`. Model names _must include the database schema name_. Some possible inputs include:
 
-1. **Data tests** check that our assumptions about our raw data are correct
+- `default.vw_pin_sale` - Rebuild a single view
+- `default.vw_pin_sale default.vw_pin_universe` - Rebuild two views at once
+- `+default.vw_pin_history` - Rebuild a view and all its upstream dependencies
+- `location.*` - Rebuild all views under the `location` schema
+- `path:models` - Rebuild the full DAG (:warning: takes a long time!)
+
+For more possible inputs using dbt node selection, see the [documentation site](https://docs.getdbt.com/reference/node-selection/syntax#examples).
```

Comment:
This section shouldn't have changed at all, I just moved it from a different location.